app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/instancesListInfo/instancesListItem/instancesListItem.jsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { projectIdSelector } from 'controllers/pages';
+import { activeProjectSelector } from 'controllers/user';
+import { URLS } from 'common/urls';
+import Parser from 'html-react-parser';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ArrowIcon from 'common/img/arrow-right-inline.svg';
+import styles from './instancesListItem.scss';
+
+const cx = classNames.bind(styles);
+
+export const InstancesListItem = ({ id, title, creator, creationInfo, disabled, onArrowClick }) => {
+  const [connected, setConnected] = useState(true);
+  const projectId = useSelector(projectIdSelector);
+  const activeProject = useSelector(activeProjectSelector);
+  function testIntegrationConnection() {
```

Comment:
I think to be more consistent it would be better to use here arrow func, like this const testInt.... => fetch(.....)
And in useEffect we can use Promise.allsettelted to handle all requests in parallel.
And I think, that logic here is work incorrectly.
I got 401 error in console but error message does not appear.